server/src/notes/notes.model.ts

```diff
@@ -0,0 +1,6 @@
+export class Note {
+    constructor(
+        public id: string,
+        public body: string, 
```

Comment:
In the `Note` class, you shouldn't have body field, because it's part of response interface. There is should be `title`, `description`, `author` etc.